guide/src/metadata.md

```diff
@@ -0,0 +1,33 @@
+# Python Project Metadata
+
+maturin supports [PEP 621](https://www.python.org/dev/peps/pep-0621/),
+you can specify python package metadata in `pyproject.toml`.
+maturin merges metadata from `Cargo.toml` and `pyproject.toml`, `pyproject.toml` take precedence over `Cargo.toml`.
+
+## Add Python dependencies
+
+To specify python dependencies, add a list `dependencies` in a `[project]` section in the `pyproject.toml`. This list is equivalent to `install_requires` in setuptools:
+
+```toml
+[project]
+dependencies = ["flask~=1.1.0", "toml==0.10.0"]
+```
+
+## Add console scripts
+
+Pip allows adding so called console scripts, which are shell commands that execute some function in you program. You can add console scripts in a section `[project.scripts]`.
```

Comment:
Strictly speaking, is this `pip` or `setuptools`?

```suggestion
Pip allows adding "console scripts", which are shell commands that execute some function in your program. You can add console scripts in a section `[project.scripts]`.
```

Reply:
`setuptools` I think? Since you can get it with just `python setup.py install`.